### PR TITLE
Add CLI flag to combine influx tags into a JSONb column

### PR DESCRIPTION
**Pull request by blagojts**

### CODE REVIEW

File: cmd/outflux/migrate_i_test.go [hunk 1]
```diff
@@ -34,6 +34,7 @@ func TestMigrateSingleValue(t *testing.T) {
 	errs := migrate(appContext, connConf, config)
 	if errs != nil {
 		t.Error(errs[0])
+		return
```
Comment (RobAtticus) [inline cmd/outflux/migrate_i_test.go]: Seems like you just want `t.Fatal` for all of these then?

File: cmd/outflux/migrate_i_test.go [hunk 2]
```diff
@@ -42,18 +43,21 @@ func TestMigrateSingleValue(t *testing.T) {
...
 		t.Error(err)
+		return
 	}
 	defer rows.Close()
 
 	var time time.Time
 	var field1 int
 	if !rows.Next() {
 		t.Fatal("couldn't check state of TS DB")
+		return
```
Comment (RobAtticus) [inline cmd/outflux/migrate_i_test.go]: Is this necessary? I thought Fatal basically ends program execution?

File: cmd/outflux/migrate_i_test.go (hunk 1 above)
Comment (blagojts) [reply cmd/outflux/migrate_i_test.go]: Will change

File: cmd/outflux/migrate_i_test.go (hunk 2 above)
Comment (blagojts) [reply cmd/outflux/migrate_i_test.go]: It's not, double checked

**Pull request merged**